test/e2e/infra/vagrant/push_antrea.sh

```diff
@@ -97,27 +105,20 @@ function pushImgToNodes() {
     docker save -o $SAVED_IMG $IMG_NAME
 
     echo "Copying $IMG_NAME image to every node..."
-    # Copy image to control-plane node
-    scp -F ssh-config $SAVED_IMG k8s-node-control-plane:/tmp/image.tar &
-    pids[0]=$!
-    # Loop over all worker nodes and copy image to each one
-    for ((i=1; i<=$NUM_WORKERS; i++)); do
-        name="k8s-node-worker-$i"
+    pids=()
+    for name in ${ALL_HOSTS[@]}; do
```

Comment:
Do we need to add a double quote around ${ALL_HOSTS[@]}? https://github.com/koalaman/shellcheck/wiki/SC2068

Reply:
Done, it should be equivalent in that case but thanks for pointing it out